studio/src/hooks/useInputFocusDetection.ts

```diff
@@ -19,11 +16,12 @@ export function useInputFocusDetection() {
       }
     };
 
-    document.addEventListener("focus", handleFocus, { capture: true, signal });
-    document.addEventListener("blur", handleBlur, { capture: true, signal });
+    document.addEventListener("focus", handleFocus, { capture: true });
+    document.addEventListener("focus", handleFocus, { capture: true });
```

Comment:
looks like the focus handler is registered twice here instead of also registering the `blur` handler